src/helpers.js

```diff
@@ -155,3 +155,99 @@ export const isRectSelected = rect => selection => {
 
   return true;
 };
+
+export const transpose = m => m[0].map((x, i) => m.map(x => x[i]));
+
+export const convertTableNodeInArrayOfRows = tableNode => {
+  const map = TableMap.get(tableNode);
+  const rows = [];
+  for (let rowIndex = 0; rowIndex < map.height; rowIndex++) {
+    const rowCells = [];
+    const seen = {};
+
+    for (let colIndex = 0; colIndex < map.width; colIndex++) {
+      const cellPos = map.map[rowIndex * map.width + colIndex];
+      const cell = tableNode.nodeAt(cellPos);
+      const rect = map.findCell(cellPos);
+      if (seen[cellPos] || rect.top !== rowIndex) {
+        rowCells.push(null);
+        continue;
+      }
+      seen[cellPos] = true;
+
+      rowCells.push(cell);
+    }
+
+    rows.push(rowCells);
+  }
+
+  return rows;
+};
+
+export const convertArrayOfRowsInTableNode = (tableNode, tableArray) => {
```

Comment:
1. convertArrayOfRows**To**TableNode
2. a comment with types and description would be helpful
3. `tableArray` should we be consistent with the name of the function? something like `arrayOfRows` would fit better here